app/javascript/components/taggingWrapper.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { TaggingWithButtonsConnected, taggingApp } from '@manageiq/react-ui-components/dist/tagging';
+import '@manageiq/react-ui-components/dist/tagging.css';
+
+class TaggingWrapper extends React.Component{
+  componentDidMount()  {
+    this.loadState(this.props.tags);
+  }
+
+  constructor(props) {
+    super(props);
+    ManageIQ.redux.addReducer(taggingApp);
+  }
+
+  loadState = (state) => this.props.loadState(state);
+  reset = () =>  this.props.reset();
+  isLoaded = () => this.props.isLoaded();
+
+  render() {
+    if (!this.props.isLoaded) return <div>bkjfbkjnkjfdkjfkj</div>
```

Comment:
Maye add some actual Loader :smile:. You can use [patternfly spinner](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Loading=true&knob-Size=md&selectedKind=patternfly-react%2FWidgets%2FSpinner&selectedStory=Spinner&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs).